docs/common-reference-docs/3-configuring-rules.md

```diff
@@ -60,20 +60,64 @@ Collab.Land supports many fungible and non-fungible tokens on many chains. Pleas
 
 ### Token Details Explained:
 
-To find the token details, please refer to this article: [Find Token Details](./4-finding-token-details)
+To find the token details, please refer to this article: [Find Token Details](./finding-token-details)
 
 **chainId***: Enter the chain ID that your token is on.The chainID is a unique ID assigned to each chain(L1 and L2) - For example 1 is the the chainID for Ethereum Mainnet 137 is the chainID of Polygon Mainnet. You can find the chainIds here: https://chainlist.org/
 
-**Type***: Select the token type of your token from the dropdown list.
+**type***: Select the token type of your token. For example, for EVM based chains, it can ERC-20, ERC-721, ERC-1155, etc.
+
+**contractAddress***: Input the contract address of your token.
+
+**tokenId (optional)**: For NFT token types such as ERC-721 & ERC-1155(Token ID Required for this token type), you can add specific token IDs.
+
+*Note*: When using the ERC-1155 token type, you must enter the token IDs for all tokens in your collection in the token ID section. Token IDs should be separated by one comma, and no spaces.
+
+**minToken (optional)**: Input the minimum amount of tokens that are required to get the role.
+
+*Note*: If you do not enter a minimum amount of tokens, the default is 1.
 
-**contractAddress***: Input the contract address of your token
+**maxToken (optional)**: Input the maximum amount of tokens that are required to get the role.
 
-Note: When using the ERC-1155 token type, you must enter the token IDs for all tokens in your collection in the token ID section. Token IDs should be separated by one comma, and no spaces.
+*Note*: If you do not enter a maximum amount of tokens, the default is infinity.
 
-Balance: Input the minimum amount, and maximum (optional) amount of tokens that are required to get the role.
+**roleID (optional)**: Role ID is a unique identifier for the rules. You can use this to identify the rules in the API response.
 
-Attributes (optional): For NFT token types such as ERC-721, you can add specific metadata conditions. If your NFT contains metadata traits and values you can enter them by clicking on the Attributes button, and clicking on + Add metadata. 
+**Attributes (optional)**: For NFT token types such as ERC-721, you can add specific metadata conditions. For example, you can add a condition that the token must have a specific attribute or trait.
 
```

Comment:
We should probably add a note here that tells the readers that we have more TGR properties and they can find them here https://api.collab.land/explorer/#/AccessControlController/AccessControlController.checkRoles